internal/runners/deploy/uninstall/uninstall.go

```diff
@@ -66,19 +76,35 @@ func (u *Uninstall) Run(params *Params) error {
 	}
 
 	logging.Debug("Attempting to uninstall deployment at %s", path)
-	if !store.New(path).HasMarker() {
+	store := store.New(path)
+	if !store.HasMarker() {
 		return errs.AddTips(
 			locale.NewError("err_deploy_uninstall_not_deployed", "There is no deployed runtime at '{{.V0}}' to uninstall.", path),
 			locale.Tl("err_deploy_uninstall_not_deployed_tip", "Either change the current directory to a deployment or supply '--path <path>' arguments."))
 	}
 
+	contents, err := fileutils.ReadFile(store.MarkerFile())
+	if err != nil {
+		return locale.WrapError(err, "err_deploy_uninstall_marker", "Unable to read marker file at [ACTIONABLE]{{.V0}}[RESET]. Deployment may be corrupted.", store.MarkerFile())
+	}
+
+	var namespace string
+	var commitID string
+	lines := strings.Split(string(contents), "\n")
```

Comment:
I'm wondering if maybe we can put this logic into a method in `store` so that we don't have to do any interpretetation of the store outside of the store package. I'm thinking if we modify the store's marker file down the road, we might have issues here that we didn't foresee.